.github/workflows/stale-issues-and-pr.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PR'
+
+# **What it does**: Mark issues and pull requests after no updates for 60 days. After that only issues will be automatically closed after 7 days of inactivity.
+# **Why we have it**: We want to manage our queue of issues and pull requests.
+# **Who does it impact**: Everyone collaborating on the project.
+
+on:
+    schedule:
+        -   cron: '10 0 * * *'
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/stale@v8
+                with:
+                    days-before-stale: 30
```

Comment:
On the docs above it says 60 days but here is 30, is here wrong or are the docs above misleading? 🤔

Reply:
fixed and added comments